.evergreen/run-tests.sh

```diff
@@ -26,7 +26,7 @@ echo "Running $AUTH tests, connecting to $MONGODB_URI"
 # Run the tests, and store the results in a Evergreen compatible JSON results file
 case "$OS" in
    *)
-      TEST_PHP_ARGS="-q -x --show-diff -g FAIL,XFAIL,BORK,WARN,LEAK,SKIP" make test TESTS=$TESTS
+      TEST_PHP_ARGS="-q -x --show-diff -g FAIL,BORK,WARN,LEAK,SKIP" make test TESTS=$TESTS
```

Comment:
Why was XFAIL removed? Is this just to suppress them from output?

Will this prevent us from noticing when an XFAIL test unexpectedly passes?

Reply:
I removed this to no longer get the diff printed for tests that are expected to fail, but it doesn't achieve what I wanted. Will revert and revisit.

For the record, tests that have an `XFAIL` section but don't fail are printed as warnings:
```
=====================================================================
WARNED TEST SUMMARY
---------------------------------------------------------------------
Test xfail [tests/xfail-pass.phpt] (warn: XFAIL section but test passes)
=====================================================================
```

Reply:
If a passing XFAIL test yields a warning, then doesn't this work as intended?

Reply:
Sorry, I was referring to the `--show-diff` portion not working as intended. Currently, a successful XFAIL test will print its diff output. I was hoping that removing the `XFAIL` portion would remove this, as I'm not interested in a diff from a test that's expected to fail anyways. Even then, a diff is shown, meaning that I might as well undo this change for now until I figure out how to accomplish this:
```
$ TEST_PHP_ARGS="-q -x --show-diff -g FAIL,BORK,WARN,LEAK,SKIP" make test TESTS=tests/connect/standalone-plain-0001.phpt

[...]
=====================================================================
Running selected tests.

========DIFF========
001+ Notice: Undefined index: port in /Users/alcaeus/Code/mongodb/mongo-php-driver/tests/connect/standalone-plain-0001.php on line 9
002+ 
003+ Fatal error: Uncaught MongoDB\Driver\Exception\InvalidArgumentException: Failed to parse MongoDB URI: 'mongodb://root:toor@127.0.0.1:0/admin'. Invalid host string in URI. in /Users/alcaeus/Code/mongodb/mongo-php-driver/tests/connect/standalone-plain-0001.php:10
004+ Stack trace:
001- User Created
002- string(9) "important"
003- User deleted
004- ===DONE===
005+ #0 /Users/alcaeus/Code/mongodb/mongo-php-driver/tests/connect/standalone-plain-0001.php(10): MongoDB\Driver\Manager->__construct('mongodb://root:...')
006+ #1 {main}
007+   thrown in /Users/alcaeus/Code/mongodb/mongo-php-driver/tests/connect/standalone-plain-0001.php on line 10
========DONE========
=====================================================================
Number of tests :    1                 1
Tests skipped   :    0 (  0.0%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :    0 (  0.0%) (  0.0%)
Expected fail   :    1 (100.0%) (100.0%)
Tests passed    :    0 (  0.0%) (  0.0%)
---------------------------------------------------------------------
Time taken      :    0 seconds
=====================================================================

=====================================================================
EXPECTED FAILED TEST SUMMARY
---------------------------------------------------------------------
Connect to MongoDB with using PLAIN auth mechanism [tests/connect/standalone-plain-0001.phpt]  XFAIL REASON: authMechanism=PLAIN (LDAP) tests must be reimplemented (PHPC-1172)
parse_url() tests must be reimplemented (PHPC-1177)
=====================================================================
```

Reply:
I reverted this change entirely.